source/nodejs/adaptivecards-site/workbox-config.json

```diff
@@ -8,7 +8,6 @@
 		"samples/*.html",
 		"schemas/**",
 		"**/*.css",
-		"designer/*",
```

Comment:
@paulcam206 I believe removing `designer/*` from the precache resolved the hashing issue we were talking about. Are there any other assets here that you think should be removed from the precache as well?

Reply:
I can't think of anything else here :)